### ISSUE TITLE
set up the initial experiments

### ISSUE THREAD

**Issue by c-schicho** [c-schicho/ZeroInitialization#2]
set up the initial experiment framework for the mnist dataset.

- [ ] FNN network with initialization
- [ ] dataloader
- [ ] train loop
- [ ] visualize training and network parameters (especially gradients)
- [ ] try for different set up variations
